config/viper_config.go

```diff
@@ -73,6 +73,7 @@ func (c *Config) fillDefaultValues() {
 		"pitaya.cluster.rpc.client.nats.pinginterval":           pitayaConfig.Cluster.RPC.Client.Nats.PingInterval,
 		"pitaya.cluster.rpc.client.nats.maxpingsoutstanding":    pitayaConfig.Cluster.RPC.Client.Nats.MaxPingsOutstanding,
 		"pitaya.cluster.rpc.client.nats.requesttimeout":         pitayaConfig.Cluster.RPC.Client.Nats.RequestTimeout,
+		"pitaya.cluster.rpc.server.loopbackenabled":             pitayaConfig.Cluster.RPC.Server.LoopbackEnabled,
```

Comment:
can you add documentation in configuration.rst for that new config and its default value?